prov/shm/src/smr_dsa.c

```diff
@@ -35,6 +35,7 @@
 #endif /* HAVE_CONFIG_H */
 
 #include "smr.h"
+#include <immintrin.h> // _mm_pause
```

Comment:
Please move this line to a place after `#if SHH_HAVE_DSA`, see the AWS CI failure when building on ARM.

Reply:
Done. Pushed this update.